ushahidi/platform-client-mzima
# [Outreachy Task Submission] Added a success message once a collection is successfully added

# Engineers Checklist

## What this PR does?
-Enhances the user experience by implementing a success message that confirms the successful addition and saving of a collection.

## How can it be tested?
- Launch the local server: http://localhost:4200
- Login/signup to the platform
- Navigate to the collection section on the sidebar
- Pop up modal opens up, click on "add collection" button
- Form is displayed, fill in the form and "Save" it


# Where is it documented?
- It is documented on an issue I raised: https://github.com/ushahidi/platform/issues/4826

## What would be the Impact of the solution?
- Implementing a success message upon adding and saving a collection would prevent users from creating duplicate collections and eliminate the need for manual confirmation of the collection list, improving user experience and increasing efficiency.

## UI Update
### The confirmation message once "Save" button is clicked

![Group 9](https://github.com/ushahidi/platform-client-mzima/assets/124133577/5c3f8139-a2d6-4c05-8a2f-0985b74f1e5f)

[Screencast from 03-14-202
4 09:59:09 AM.webm](https://github.com/ushahidi/platform-client-mzima/assets/124133577/b08f79e8-85a9-49fc-b0ea-81ab13f876f1)
